fase_di_sviluppo/src/modules/calendar.js

```diff
@@ -1,26 +1,92 @@
-/**
- * Implements the Calendar class.
- * 
- * This file implements the class Calendar, which
- * represent a list of homework.
- * 
- * @file This file defines Calendar class.
- * @author Andrea Lanzani
- * Everyone who modifies this file MUST add his/her name
- * @version 1.0alpha
- */
-
-/**
- * @todo Implement this class.
- * @todo Write the documentation.
- * 
- * Class representing a calendar.
- */
-export class Calendar {
-  /**
-   * Create a calendar.
-   */
-  constructor() {
-
-  }
+/**
+ * Implements the Calendar class.
+ * 
+ * This file implements the class Calendar, which
+ * represent a list of homework.
+ * 
+ * @file This file defines Calendar class.
+ * @author Frabcesco Colombo
+ * Everyone who modifies this file MUST add his/her name
+ * @version 1.0alpha
+ */
+
+/**
+ * @todo Implement this class.
+ * @todo Write the documentation.
+ * 
+ * Class representing a calendar.
+ */
+
+export class Calendar {
+
+  private homework: Homework[];
+  private vettElement : number;
+
+  constructor() {
+        this.homework = null;
+        this.vettElement = 0;
+  }
+
+  getNumeberElements(){
+        return this.vettElement;
+  }
+
+  addHomework(homework : Homework){
+      
+    if(homework.duration.isBusy()){
+        this.homework.push(homework);
+        this.vettElement++;
+    }
+    else
+        window.alert("Could not enter assignment");
+  }

```

Comment:
L'if è invertito: se il giorno è occupato dovresti mandare l'alert. Il fix è molto semplice: aggiungi un not (!) all'if
```suggestion
    if(!homework.duration.isBusy()){
        this.homework.push(homework);
        this.vettElement++;
    }
    else {
        window.alert("Could not enter assignment");
    }
  }
```

Per favore l'uso delle parentesi graffe è obbligatorio anche per operazioni da una riga. Questo è giusto per mantenere uno stile uguale in tutti i file.